test/commands/generate.test.js

```diff
@@ -19,7 +19,8 @@ describe('feathers-cli', () => {
         'feathers:hook',
         'feathers:middleware',
         'feathers:model',
-        'feathers:service'
+        'feathers:service',
+        'feathers-plugin'
```

Comment:
I would probably just call it `feathers:plugin` so that it follows the same naming convention. This one is a bit different however because that generator sort of assumes that the plugin will be published to npm.